torchvision/prototype/transforms/_utils.py

```diff
@@ -0,0 +1,19 @@
+from typing import Any, Union, Optional
+
+import PIL.Image
+import torch
+from torchvision.prototype import features
+from torchvision.prototype.utils._internal import query_recursively
+
+
+def query_image(sample: Any) -> Union[PIL.Image.Image, torch.Tensor, features.Image]:
```

Comment:
Nit: Shall we put it next to `query_recursively`?

Reply:
That would mean that `torchvision.prototype.utils` depends on `torchvision.prototype.features`. Just checking if we want that, because right now `torchvision.utils` is independent of the other submodules. `torchvision.prototype.{datasets, transforms}` need to depend on `torchvision.prototype.features` anyway, so we don't need to take this into consideration there.

Reply:
Sounds good. Let's not block and decide later.